setup.py

```diff
@@ -1,7 +1,15 @@
+
 import setuptools
-from pip._internal.req import parse_requirements
 
-install_reqs = parse_requirements('requirements.txt', session='hack')
+install_reqs = ["fire",
```

Comment:
nope, this should stay as it was before - probably it was changed back because of some mistake